### DISCUSSION TITLE
Feedback and issues with the Installation / Usage of docmost

### DISCUSSION THREAD

**Discussion by proxforge** [docmost/docmost#144] [Help / Questions] [answered]
I already posted a feedback [here](https://www.reddit.com/r/selfhosted/comments/1drd01d/comment/lfuyjiu/). I also reinstalled from scratch with having the domain set correctly in the first place. 

If I open docmost on Windows 10 Chrome via https://docs.proxforge.de and login with my credentials and create a page it gets created. If I open a browser on my mobile phone and login to https://docs.proxforge.de I can only see the title of the create page, but no content.

Edit: This also happens vice versa. If I create something on mobile, only the title is there on desktop.

**Comment by Philipinho** [comment]
Can you confirm if the /collab websocket path transmits data when you type (on your browser console)?

**Reply by proxforge** [reply]
![image](https://github.com/user-attachments/assets/24d44c7c-c337-413a-a453-519516623a39)

I dont know what this means but this comes when I open up the developer console on chrome.

**Reply by proxforge** [reply]
Thanks! You pointed me in the right direction. Had to to websocket support in nginx reverse proxy, after enabling it works! THANKS!

![image](https://github.com/user-attachments/assets/9ad70a92-4b76-4554-89b8-e7c513e45531)

**Reply by Philipinho** [reply, marked as answer]
From your initial reddit comment, I take it that you are using NGINX reverse.

If not there already, you should add this to your NGINX `location` block to make the websocket work.
```
proxy_set_header Upgrade $http_upgrade;
proxy_set_header Connection "upgrade";
```

**Reply by Philipinho** [reply]
Glad you figured it out. I was already typing my response.

**Reply by proxforge** [reply]
Thanks, will update my reddit post, so the feedback is correct. Seems like everything that did not work before, works now (or at least most, havent tried out all)
